src/chips/nuked_opl3_v174.cpp

```diff
@@ -87,7 +88,7 @@ int NukedOPL3v174::generateAndMix32(int32_t *output, size_t frames)
     return (int)frames;
 }
 
-const char *NukedOPL3v174::emulatorName()
+int NukedOPL3v174::emulatorId()
 {
-    return "Nuked OPL3 (v 1.7.4)";
+    return ADLMIDI_EMU_NUKED_174;
```

Comment:
It's BAD idea to put dependency of chipset from ADLMIDI as I told, I use same chipset in both ADLMIDI and OPL3 Bank Editor where is completely different base. You better duplicate the stuff into chipset too, to make it be independent. And yeah, Don't break `emulatorName()` as it is mainly using in OPL3 Bank Editor and is more convenient to call it rather request entry from separate array.